NeosModLoader/ModLoader.cs

```diff
@@ -39,58 +39,16 @@ internal static void LoadMods()
             }
             SplashChanger.SetCustom("Looking for mods");
 
-            string modDirectory = Path.Combine(Directory.GetCurrentDirectory(), "nml_mods");
-
-            Logger.MsgInternal($"loading mods from {modDirectory}");
-
             // generate list of assemblies to load
-            ModAssembly[] modsToLoad = null;
-            try
-            {
-                modsToLoad = Directory.GetFiles(modDirectory, "*.dll")
-                    .Select(file => new ModAssembly(file))
-                    .ToArray();
-
-                Array.Sort(modsToLoad, (a, b) => string.CompareOrdinal(a.File, b.File));
-            }
-            catch (Exception e)
-            {
-                if (e is DirectoryNotFoundException)
-                {
-                    Logger.MsgInternal("mod directory not found, creating it now.");
-                    try
-                    {
-                        Directory.CreateDirectory(modDirectory);
-                    }
-                    catch (Exception e2)
-                    {
-                        Logger.ErrorInternal($"Error creating mod directory:\n{e2}");
-                    }
-                }
-                else
-                {
-                    Logger.ErrorInternal($"Error enumerating mod directory:\n{e}");
-                }
-                return;
-            }
+            AssemblyFile[] modsToLoad;
+            if (AssemblyLoader.LoadAssembliesFromDir("nml_mods") is AssemblyFile[] arr) {
+                modsToLoad = arr;
+            } else return;
```

Comment:
While I personally like the whole { on same line style, it seems like the standard C# style is to do it on a newline like a barbarian. Can you switch this to { on newlines?